frontend/src/components/CreateLeague/CreateLeague.js

```diff
@@ -34,8 +34,7 @@ function CreateLeague() {
 
     return (
         <div>
-            <div>
-               <div className="create-league-form-title"></div>
+            <div className="create-league-form-title">
                 <h2 className="order-title">Create your fantasy league</h2>
```

Comment:
I resolved this incorrectly. There should be no h2 tag it should just be the text in the `create-league-form-title` div

Reply:
I made the change based on what you said @JScheire let me know if it's incorrect and I'll change it.

Reply:
Yeah I completely forgot to push the changes I made after I resolved, so that's why it wasn't working